scipy/optimize/tests/test__linprog_clean_inputs.py

```diff
@@ -6,58 +6,58 @@
 import numpy as np
 from numpy.testing import assert_, assert_allclose
 from pytest import raises as assert_raises
-from scipy.optimize._linprog_util import _clean_inputs
+from scipy.optimize._linprog_util import _clean_inputs, _LPProblem
 from copy import deepcopy
 
 
 def test_aliasing():
-    c = 1
-    A_ub = [[1]]
-    b_ub = [1]
-    A_eq = [[1]]
-    b_eq = [1]
-    bounds = (-np.inf, np.inf)
-
-    c_copy = deepcopy(c)
-    A_ub_copy = deepcopy(A_ub)
-    b_ub_copy = deepcopy(b_ub)
-    A_eq_copy = deepcopy(A_eq)
-    b_eq_copy = deepcopy(b_eq)
-    bounds_copy = deepcopy(bounds)
-
-    _clean_inputs(c, A_ub, b_ub, A_eq, b_eq, bounds)
-
-    assert_(c == c_copy, "c modified by _clean_inputs")
-    assert_(A_ub == A_ub_copy, "A_ub modified by _clean_inputs")
-    assert_(b_ub == b_ub_copy, "b_ub modified by _clean_inputs")
-    assert_(A_eq == A_eq_copy, "A_eq modified by _clean_inputs")
-    assert_(b_eq == b_eq_copy, "b_eq modified by _clean_inputs")
-    assert_(bounds == bounds_copy, "bounds modified by _clean_inputs")
+    """
+    Test for ensuring that no objects referred to by `lp` attributes,
+    `c`, `A_ub`, `b_ub`, `A_eq`, `b_eq`, `bounds`, have been modified
+    by `_clean_inputs` as a side effect.
```

Comment:
The blank character that separates `'a'` and `'side'` is a "no-break space".  That's Unicode code point U+00A0, and encoded in UTF-8, it is the two bytes `b'\xc2\xa0'`.  This should be changed to an ASCII space.